hack/test-end-to-end-scenario.sh

```diff
@@ -25,6 +26,7 @@ function wait_for_app() {
 
 	echo "[INFO] Waiting for frontend pod to start"
 	wait_for_command "oc get -n $1 pods | grep frontend | grep -i Running" $((120*TIME_SEC))
+	oc deploy database --logs -n $1
```

Comment:
As above: there's no guarantee these logs will still be available if the deployment was successful.